src/multi.jl

```diff
@@ -9,45 +9,131 @@ function amp_post_convhull(m::PODNonlinearModel; kwargs...)
     # Variable holders
     λ = Dict()  # Extreme points and multipliers
     α = Dict()  # Partitioning Variables
+    β = Dict()  # Lifted variables for further
```

Comment:
Lifted variables for further what - complete the comment

Reply:
This is updated for the most recent commit.